CHANGELOG.md

```diff
@@ -147,7 +147,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     * Each of the training loops now keeps its own results collection ([#7891](https://github.com/PyTorchLightning/pytorch-lightning/pull/7891))
     * Remove `EpochResultStore` and `HookResultStore` in favor of `ResultCollection` ([#7909](https://github.com/PyTorchLightning/pytorch-lightning/pull/7909))
     * Remove `MetricsHolder` ([#7909](https://github.com/PyTorchLightning/pytorch-lightning/pull/7909))
-
+    * Add `load_state_dict` to ResultCollection ([#7948](https://github.com/PyTorchLightning/pytorch-lightning/pull/7948))
+    * Add `result_collections` to checkpoint and `restore_result_collections` to `CheckpointConnector` ([#7966](https://github.com/PyTorchLightning/pytorch-lightning/pull/7966))
```

Comment:
I added a section for "Fault-tolerant training" (line 83)

Reply:
Done !